.github/workflows/build-container-and-test.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   lint:
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9, 3.10.1]
```

Comment:
Why you include the minor? Same for the matrix line below

Reply:
See commit message of https://github.com/Granulate/gprofiler/pull/255/commits/659d4f58d869522cfa980e869442f53ece68a829 (and see the CI status of https://github.com/Granulate/gprofiler/pull/255/commits/2254671565a76bba40d8c109edb6ea562bcb3da7, the previous commit)

Reply:
WTF lol